app/services/proofing/aamva/request/verification_request.rb

```diff
@@ -84,9 +84,36 @@ def add_user_provided_data_to_body
             inside: '//dldv:verifyDriverLicenseDataRequest',
           )
 
+          if IdentityConfig.store.aamva_send_id_type
+            add_state_id_type(
+              applicant.state_id_data.state_id_type,
+              document,
+            )
+          end
+
           @body = document.to_s
         end
 
+        def add_state_id_type(id_type, document)
+          category_code = case id_type
+                          when 'drivers_license'
+                            1
+                          when 'drivers_permit'
+                            2
+                          when 'state_id_card'
+                            3
+                          end
+
```

Comment:
(suggestion, non-blocking): It would be great if this was stored as a `const` at the top of the class. 
(:wishes fruitlessly that Ruby supported enums:)